source/css/_base.text.scss

```diff
@@ -32,18 +32,20 @@ blockquote {
   p {
     font-size: rem(24);
     line-height: 1.3;
-    display: inline;
+    margin-bottom: 1em;
```

Comment:
I think if you don't add this `margin-bottom` it would fall back to the standard bottom margin, which I think _is_ about `1em`. Also, if you want a custom value here, could you use `rem()`? Just for code consistency sake.